.travis.yml

```diff
@@ -1,73 +1,82 @@
+      sudo docker pull welder/web-nodejs:latest
+      sudo docker pull welder/web:latest
+      sudo docker pull welder/web-e2e-tests:latest
+  else
+      npm install
+  fi
 env:
   global:
-    secure: "Q9X11ehcl4YE0ylxDGk8NgVeO7nIpqWiI9am2GA6JVXcxeYYcLf3NHGe1JiLwFQ/xSq4KJrY0IgpivkVrPwTOojpR/tRG5QNDR/IJzN/6PV0IAWkHjPDKZexCQmy4IoPFib/cWqxo46V/JZ+gzs/TJugkl5/QpTcGlCr+d7VkuA80Bj0nlQO5GevYVgYMCrN6u6cKNOWtNQ0gppxHrpsLo36MpHqHbNbv236GPGsV4K2qpiqvvO5ogD+LCv5xoKEZK0ext4FNrrZFq6XEYdy9uUaUbSFfoG+HXESXbIt0L7krQDT/PkKI0X4nM91eYl5mS63+HVue7vLiijS5h9+YCa8HblMIiAiyrR3lPPH0Hx4FVSrQfkvtoXMBs3nRt7keu2FG370a7WoaQ71Q2jHrI9cfYzRwa/Mok73eF/oeN32hPlPDD679hWOZYqbSksSTikCLIgwJpEgIfaHgEYXFmL2JFXk0HbgIciA+C9fE0rfaUbmARtWYq8c+b52aIxAHEV9zYy3JeARrYxk7s3jQgS/LZIw4rfso+bQIvEMUDWt8MnvKaecY1E7XjOpW1kuk8gVjZbHm6oHKl2IndZA9KNpXjT2ew8d3SB69l2lswUYu4mTdywNEUt6nIaQtROiiftUKYThAbibIEIrq7GxQSOYMUPbvUYsZO5rkVssJps="
+  - secure: Q9X11ehcl4YE0ylxDGk8NgVeO7nIpqWiI9am2GA6JVXcxeYYcLf3NHGe1JiLwFQ/xSq4KJrY0IgpivkVrPwTOojpR/tRG5QNDR/IJzN/6PV0IAWkHjPDKZexCQmy4IoPFib/cWqxo46V/JZ+gzs/TJugkl5/QpTcGlCr+d7VkuA80Bj0nlQO5GevYVgYMCrN6u6cKNOWtNQ0gppxHrpsLo36MpHqHbNbv236GPGsV4K2qpiqvvO5ogD+LCv5xoKEZK0ext4FNrrZFq6XEYdy9uUaUbSFfoG+HXESXbIt0L7krQDT/PkKI0X4nM91eYl5mS63+HVue7vLiijS5h9+YCa8HblMIiAiyrR3lPPH0Hx4FVSrQfkvtoXMBs3nRt7keu2FG370a7WoaQ71Q2jHrI9cfYzRwa/Mok73eF/oeN32hPlPDD679hWOZYqbSksSTikCLIgwJpEgIfaHgEYXFmL2JFXk0HbgIciA+C9fE0rfaUbmARtWYq8c+b52aIxAHEV9zYy3JeARrYxk7s3jQgS/LZIw4rfso+bQIvEMUDWt8MnvKaecY1E7XjOpW1kuk8gVjZbHm6oHKl2IndZA9KNpXjT2ew8d3SB69l2lswUYu4mTdywNEUt6nIaQtROiiftUKYThAbibIEIrq7GxQSOYMUPbvUYsZO5rkVssJps=
+  - secure: K5mPGLWUFqX/Q0aNoUgYOONB9WKkd98FyMRX5UQQS2Q/lT4/OZWPezQ1I81ufK53r4VLXspL/EILNFXPHb8MEh5CfiIutL/39c/fAFY7XR9EzsuR/DdyIhuWRG4YhqlMU+g2F3zXqcZGgkrLQlvRlCsUp23DBBKZA76EgARMyg4TwVoNbd8vNRwuuo6qIh9d7YT2WcXv/fkF+lH36rjViC2TPFOmgOTkkdumLw/xCxOWK3IBXIPP295pq+Uo/kk6rXUvKrB4Feg2mBd+knvHVVKeu4RDqn2gOz6StP6z085mtr4EhMG4DMT1rGG9KoYnOIaH+cPBCtII+z0Snl3BXCEINR4CpyrrIuJEI5GGWNohr52KaN4i/5eS7Fn6M6ilTB0ohzvSCBEjl5EcV764yHVZJ69hKEBg4Goch17jwXL8vdFB8fPEWBqmAo7PBwXpbfTZCGgHJrpUpbAeJj77IZ7CFx2nOZSe1tsqLP7eCj7P+w9l2AHUMDoPbPstukLMkNpUpEI7ybzZCIJwWTq8KzsnjhFOPA17CF61IFKfyUD600grs2Ye7SyiVDxHoRo0IOMKhFreanwUNc9oxuVUuM/bfCpL9sm9Lxg8/bo2tGrUTb7LfKZOyi93ijHvJRCpzyf4F7otNoYFxNBHj45/xlYVO+4weKncJ3qbjAEE5DE=
+  - secure: KixSETk7puLVyxEfCfvDJHl0us1X0CR9Xzeh90sz73JlnFcduv9EaJv/r9bzfjz/dnEmsuaScHJQ8mF0gayQi4ood1oB5ywWkNDfCGhgRG2BmW9FliDU4ukQRDqJq9peNb1TOD3x1LPL4dyNO4Yq8h7pR36KVTeD4hqJfnvlvdNiJ9tk/n6LaR9AFDzkyDfAqsEDqqk/fShw7CociLTsEZ4OTtX9dP/egQm0OA8KYHGxTubKI1Qn6U/4jZn85TajVfEg9HawasjJnFGURbwmCujiB2tLaMCJ+6uflkhyeW5zBKY0zBbMiSZU9lP+enkro6M9JyckGjrBDTxQTn1l0u557wdAzn7kIsyXdwfkqabjh3bicCA9xFkREyIpzAt8yNyqXZrsKJLURz2PVwmwI9ZEpE4hNfRtt3qAK98LYrtvW1JnRbaAci79TLBEfzaLNvn7+ZWXBzsoFTDrbaU70ox3gKcTtast9JtYNcWNNyGXJY6xCRYo9qPJ6vDkQnplA026zioP8q+Kuq2l5WRs4HmtoXxbmJpGya1dVgH/8giG7VU7Xf9q/7bW7e6ytM/UlJhheqJFC9aHBfIe53PNo8fL5XgjCFcAAXQ29i0X+ye6258rD0b9vswkkxQbo69dyI9fR/GPcndL3/VNfeZNTFG9BYoI5pwFnb5JOJoUfwo=
```

Comment:
Why do we have 2 more secure strings? What variables are in there ?

Reply:
These are AWS access key and AWS secret access key. They are required by Artifacts.

Reply:
👍